### ISSUE TITLE
Error: libloadorder failed to set the load order. Details: second time provided was later than self

### ISSUE THREAD

**Issue by LosEagle** [loot/loot#921]
Apologies in advance if I made a mistake and this issue belongs in Oblivion metadata tracker. If it does, please inform me and I will move it asap.   
   
So I ran loot on almost clean Oblivion without any mods except for Wrye's bashed patch and OBSE. It does change the load order but I can't apply it. It throws error saying that plugins.txt might be read-only but I checked \AppData\Local\Oblivion\plugins.txt and it isn't.    
   
This is what LOOTDebugLog says:    
`[08:28:44.594980] [error]: libloadorder failed to set the load order. Details: second time provided was later than self` 
`[08:28:44.595046] [error]: Exception while executing query: libloadorder failed to set the load order. Details: second time provided was later than self: Libloadorder error`
`[08:28:44.595513] [error]: Chromium console message from http://loot/ui/app.bundle.js at line 1: Error: Oh no, something went wrong! This is usually because your plugins.txt is set to be read-only. If it is, unset it and try again. If it isn't, you can check your LOOTDebugLog.txt (you can get to it through the main menu) for more information.
    at onFailure (http://loot/ui/app.bundle.js:1:140358) `
   
I tried running loot as admin and disabling UAC.   
I'm running v0.12.5

**Comment by WrinklyNinja** [comment]
Very strange... I think this only occurs if one of your plugins has a modification timestamp earlier than 1970-01-01 00:00:00, which I wasn't expecting to ever occur.

Can you upload your full log? A screenshot or screenshots of your Data folder so I can see your plugins and their "Date modified" timestamps could also be helpful.

**Comment by LosEagle** [comment]
Of course. Seems like .esp files have no date at all? 
![oblivion](https://user-images.githubusercontent.com/6146756/37861912-5636fbf6-2f44-11e8-80ba-d5f9c1c5c0b6.PNG)
[LOOTDebugLog.txt](https://github.com/loot/loot/files/1844174/LOOTDebugLog.txt)

**Comment by WrinklyNinja** [comment]
Well, I've never seen anything quite like that... What's shown if you right-click one of the plugins with no modification time and go to Properties -> Details?

I'll need to update libloadorder/LOOT API/LOOT to account for this being a possibility. I'm surprised anything else works with it - what does Wrye Bash display in its load order UI?

**Comment by LosEagle** [comment]
The dates in properties differ by days and hours but they all say january 1970 including bashed patch. I must have messed up when setting up nmm and wrye at some point. Wrye also shows interesting dates. Rip Unix time.
![wrye](https://user-images.githubusercontent.com/6146756/37861982-14c4bd28-2f46-11e8-93df-177bc408f6e8.PNG)

**Comment by WrinklyNinja** [comment]
Could you zip and upload your Bashed Patch to help me test a fix in case I can't figure out how to replicate the timestamps?

**Comment by LosEagle** [comment]
Yep. I'd also like to point out that I'm using the Wrye Bash 307 beta2 not the 304 stable version. 
[Bashed Patch, 0.zip](https://github.com/loot/loot/files/1844264/Bashed.Patch.0.zip)

**Comment by WrinklyNinja** [comment]
OK, I think I have a fix, get [this LOOT build](https://bintray.com/wrinklyninja/loot/download_file?file_path=loot_0.12.5-21-g4817b06_group-metadata.7z) and replace its `loot_api.dll` with [this one](https://bintray.com/wrinklyninja/loot/download_file?file_path=loot_api-0.12.5-14-g6bf2238_pre-unix-epoch-timestamps-win32.7z), then try sorting again.

**Comment by LosEagle** [comment]
I was able to apply the changes with that. Seems the issue is solved.

**Comment by WrinklyNinja** [comment]
I'll keep this open until the fix is merged into `dev`.
